backend/src/election/mod.rs

```diff
@@ -148,3 +150,38 @@ pub async fn election_download_results(
         .filename(filename)
         .content_type("application/pdf"))
 }
+
+/// Download a generated EML_NL 510 XML file with election results
+#[utoipa::path(
+    get,
+    path = "/api/elections/{election_id}/download_xml_results",
+    responses(
+        (
+            status = 200,
+            description = "XML",
+            content_type="text/xml",
+            headers(
+                ("Content-Type", description = "text/xml"),
+            )
```

Comment:
nitpick: does the content type need to be specified twice?

Reply:
For now I think we can just keep this as is. These two occurrences do end up on two different places in the openapi json file.